swedbank-pay-woocommerce-checkout.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/*
+ * Plugin Name: Swedbank Pay Checkout
+ * Plugin URI: https://www.swedbankpay.com/
+ * Description: (Preview). Provides a Credit Card Payment Gateway through Swedbank Pay for WooCommerce.
+ * Author: Swedbank Pay
+ * Author URI: https://profiles.wordpress.org/swedbankpay/
+ * License: Apache License 2.0
+ * License URI: http://www.apache.org/licenses/LICENSE-2.0
+ * Version: 3.0.0-beta.1
+ * Text Domain: swedbank-pay-woocommerce-checkout
+ * Domain Path: /languages
+ * WC requires at least: 3.0.0
+ * WC tested up to: 3.9.1
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+} // Exit if accessed directly
+
+if ( ! class_exists( 'WC_Swedbank_Plugin', false ) ) {
+	include_once( dirname( __FILE__ ) . '/includes/class-wc-swedbank-plugin.php' );
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function dirname() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)